src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -13,6 +13,8 @@
         <dd i18n>{{ detailsCardData.orchestrator || 'Orchestrator is not available' }}</dd>
         <dt>Ceph version</dt>
         <dd>{{ detailsCardData.cephVersion }}</dd>
+        <dt>API Documentation</dt>
```

Comment:
`Cluster API` or just ` API` might be good! but let's ask team's (@ceph/dashboard ) thought.